data/containers/bats_skip_notok.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+"""
+This script is used to selectively comment "not ok" lines in TAP files so the
+openQA TAP parser ignores them
+"""
+
+import re
+import sys
+
+
+def flush(file, lines, comment_block=False):
+    """
+    Flush buffer optionally commenting the block
+    """
+    if len(lines) == 0:
+        return
+    if comment_block:
+        # We only comment the first line as the rest have "# " in them
+        print(f"#{lines[0]}", file=file)
+        if len(lines) > 1:
+            print("\n".join(lines[1:]), file=file)
+    else:
+        print("\n".join(lines), file=file)
+
+
+def comment_not_ok(path, tests):
+    """
+    Comment "not ok" lines in path for the specified tests
+    """
+    skipped = re.compile(rf"in test file .*/(?:{'|'.join(tests)}).bats")
```

Comment:
where is used `skipped` ?

Reply:
> where is used `skipped` ?

In line 53.

To test the script, download [the .tap file linked in the description](https://openqa.opensuse.org/tests/4268394/file/podman_integration-bats-user-local.tap) and run it like this:

`./bats_skip_notok.py podman_integration-bats-user-local.tap 200-pod` (you can use any test name that appears with `grep "in test file" podman_integration-bats-user-local.tap`

As it patches the file you have to keep a copy to check with `diff`.

Reply:
oh, Im blind.